bindings/python/iota_sdk/types/node_info.py

```diff
@@ -279,6 +279,7 @@ class ProtocolParameters:
     bech32_hrp: str
     storage_score_parameters: StorageScoreParameters
     work_score_parameters: WorkScoreParameters
+    mana_parameters: ManaParameters
```

Comment:
Not like it matters but does order has an influence in Python?

Reply:
When you create the class that parameters are ordered like this and also to json will use it, apart from that optional alvalues must be last, but that's about it

This here I only noticed because I initially tried to compare the json without sorting (which then didn't work because of the optional fields)